www/js/modules/charts/controllers/HistogramChartController.js

```diff
@@ -199,19 +199,25 @@ export default class HistogramChartController {
 
         let that = this;
 
+		// must remove histogram to update histogram
+	    d3.select('#myHistogram123').remove();
+		this.$histogram = null;
```

Comment:
You have to remove all the histogram bars in order to update them.  I ran into this issue and from what I remember I think it's just one of those things when working with the ScaleBand scale in d3.